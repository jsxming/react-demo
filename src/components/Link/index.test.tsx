import Link from './index';
import {render,screen} from '@testing-library/react'
it('changes the class when hovered', () => {
  render(<Link>123</Link>)
  const linkElement = screen.getByText(/123/i);
  expect(linkElement).toMatchSnapshot()
  expect(linkElement).toBeInTheDocument();
});