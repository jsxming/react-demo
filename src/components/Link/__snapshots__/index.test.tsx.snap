// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<a
  class="normal"
  href="#"
>
  123
</a>
`;
